src/compiler/old/const.js

```diff
@@ -1,11 +1,10 @@
-(function () {
-    var Const = {};
+// --------------------------------------------------------
+// WARNING: this file is used by both by client and server.
+// Do not use any browser or node specific API!
+// --------------------------------------------------------
```

Comment:
WARNING: this file is used by both the client and the server. Do not use any browser or node-specific API!